typings/internal-ambient.d.ts

```diff
@@ -36,7 +36,16 @@ declare namespace NodeJS {
   }
 }
 
+declare module NodeJS  {
```

Comment:
This concerns me, why are we having to redeclare node globals, shouldn't they be coming through from `@types/node`

Reply:
Ah, I can answer that! `@types/node` has those defined as _actual_ globals.

Not defined:
```
global.__dirname
```

Defined:
```
__dirname
```